mindsdb/__init__.py

```diff
@@ -2,11 +2,24 @@
 if sys.version_info < (3,3):
     sys.exit('Sorry, For MindsDB Python < 3.3 is not supported')
 
+from mindsdb.config import CONFIG
+import mindsdb.libs.constants.mindsdb as CONST
 
-from .libs.data_types.data_source import DataSource
-from .libs.data_sources import *
-from .libs.controllers.mindsdb_controller import MindsDBController
+from mindsdb.libs.data_types.data_source import DataSource
+from mindsdb.libs.data_sources import *
+from mindsdb.libs.controllers.mindsdb_controller import MindsDBController
 
 name = "mindsdb"
 MindsDB = MindsDBController
-MDS = DataSource # A Mindsdb Data Source
\ No newline at end of file
+MDS = DataSource # A Mindsdb Data Source
+
+
+class Anon:
```

Comment:
Please remove this. people can access all CONFIG variables 
as 
```
from mindsdb import CONFIG
```
and all constants as 
```
from mindsdb import CONST
```